packages/syft/tests/trials/skill_test.py

```diff
@@ -16,6 +16,7 @@ def get_padawans(cohort: str) -> Dict[str, str]:
             "rodrigo": "PASSED",
             "kanak": "PASSED",
             "theresa": "PASSED",
+            "mihir":"PASSED"
```

Comment:
NIT: could you also include `,` at the end.

Reply:
Added, thanks!